src/main/resources/mybatis/collections.xml

```diff
@@ -9,6 +9,67 @@
         trunc(ST_YMax(ST_Extent(st_transform(geom, 4326)))::numeric,7) ymax
     </sql>
 
+    <sql id="featureCols">
```

Comment:
This file is getting long. Please separate the feature logic into it's own file.

Also, I didn't notice it in the previous round of changes, be careful with camel casing of sql aliases. PostgreSQL will treat them as all lowercase. This could lead to confusion in the future.